lib/matplotlib/backends/qt_editor/figureoptions.py

```diff
@@ -137,39 +137,43 @@ def prepare_data(d, init):
     # Is there a curve displayed?
     has_curve = bool(curves)
 
-    # Get / Images
-    imagedict = {}
-    for image in axes.get_images():
-        label = image.get_label()
-        if label == '_nolegend_':
+    # Get ScalarMappables.
```

Comment:
Minor formatting nitpicking. I think we would usually do `# get ScalarMappables`. Same with the `set` below.

Reply:
I capitalize all my comments and include final stops (Matplotlib or not)...